src/styles/layout.scss

```diff
@@ -107,7 +107,8 @@ a:focus {
 
 .material-icons {
   font-size: 2rem;
-  padding: 1rem;
+  padding: calc(var(--space-24) + 1rem) calc(var(--space-16) + 1rem)
```

Comment:
Changing styles of `material-icons` is breaking buttons on Download page

## Download Page
![image](https://user-images.githubusercontent.com/54291836/125656165-773deb82-c0c5-4596-ba57-03538807745c.png)
![image](https://user-images.githubusercontent.com/54291836/125656197-80631e3f-eb3e-48ae-9b7c-d78b880d7338.png)

I think you could use locally scoped classes for navigation bar